target/linux/generic/hack-5.4/991-platform-x86-pcengines-apuv2-revert-simswitch.patch

```diff
@@ -0,0 +1,32 @@
+From 8c9254d41881c81bea610193c6ac59c8cb8b79fe Mon Sep 17 00:00:00 2001
+From: Florian Eckert <fe@dev.tdt.de>
+Date: Fri, 27 Mar 2020 16:11:55 +0100
+Subject: [PATCH] Revert "platform/x86: pcengines-apuv2: wire up simswitch gpio
+ as led"
+
+This reverts commit 5037d4ddda31c2dbbb018109655f61054b1756dc.
```

Comment:
Please add a description why you reverted this.
Just put the commit message also here.